.travis.yml

```diff
@@ -25,7 +25,6 @@ before_script:
   - Xvfb :99 -screen 0 1024x768x24 -extension RANDR &
 
 script:
-  - npm run clean
```

Comment:
I had to add this in because it was uploading old builds even when new ones were created. I think we need to keep this in or we may have travis uploading incorrect builds again.